source/_integrations/syncthing.markdown

```diff
@@ -0,0 +1,42 @@
+---
+title: Syncthing
+description: Instructions on how to integrate Syncthing within Home Assistant.
+logo: syncthing.png
```

Comment:
This is old style, which we no longer use.
```
```

Additionally, please remove `source/images/supported_brands/syncthing.png`.

The new method uses our `brands` repository.